core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -9,9 +9,11 @@ import { ExecutionState } from "../../../../../workspace/types/execute-workflow.
 import { DeletePromptComponent } from "../../../delete-prompt/delete-prompt.component";
 import { NotificationService } from "../../../../../common/service/notification/notification.service";
 import Fuse from "fuse.js";
-import { filter } from "lodash";
+import { filter, iteratee } from "lodash";
```

Comment:
is this for the .join, you can remove it when you fix the logic of not splitting by spaces